packages/frontend-acceptance-tests/src/step-definitions/Newsletter/I_select_receive_newsletter_and_enter_###.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const { defineStep } = require('cucumber')
+const { core } = require('defra-wdio-core')
+const NewsletterPage = require('../../pages/newsletter')
+
+defineStep(/^I receive a newsletter and enter "(.*)"$/, function (setEmailAddress) {
+  NewsletterPage.checkUrl()
+  NewsletterPage.setNewsletterYes(setEmailAddress)
+  NewsletterPage.continue()
+})
+
+defineStep(/^I receive a newsletter and enter no email "(.*)"$/, function (setEmailAddress) {
```

Comment:
Split each step definition into its own file

You can also remove the "(.*)" from the step definition and the setEmailAddress parameter as you're never passing an email address in